Cargo.toml

```diff
@@ -107,7 +107,7 @@ aws-types = "1"
 etcd-client = { package = "madsim-etcd-client", version = "0.4" }
 futures-async-stream = "0.2.9"
 hytra = "0.1"
-rdkafka = { package = "madsim-rdkafka", version = "0.3.0", features = [
+rdkafka = { package = "madsim-rdkafka", version = "0.3.1", features = [
```

Comment:
Why to change this

Reply:
I think @wangrunji0408 does this, to support some API in madsim.